packages/genesys-spark-components/src/components/beta/gux-label-info/gux-label-info.tsx

```diff
@@ -0,0 +1,76 @@
+import { Component, Element, h, JSX, Prop, Method } from '@stencil/core';
+
+import { Placement } from '@floating-ui/dom';
+
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxLabelInfoVariant } from './gux-label-info.types';
+import { getSlotTextContent } from '../../../utils/dom/get-slot-text-content';
+
+/**
+ * @slot content - Required slot for tooltip and screenreader content
+ */
+
+@Component({
+  styleUrl: 'gux-label-info.scss',
+  tag: 'gux-label-info-beta',
+  shadow: true
+})
+export class GuxLabelInfo {
+  private tooltip: HTMLGuxTooltipBetaElement;
+
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  variant: GuxLabelInfoVariant = 'info';
+
+  @Prop({ mutable: true })
+  placement: Placement = 'right';
+
+  componentWillLoad(): void {
+    trackComponent(this.root, { variant: this.variant });
+  }
+
+  private getVariantIcon(variant: GuxLabelInfoVariant): string {
+    return variant === 'question'
+      ? 'fa/circle-question-regular'
```

Comment:
I think we hardcode FA icon names around the app. I wonder if there'd be a benefit to not hardcoding it like we do and creating an icon tsx file, or maybe that'd be overkill. Instead of `fa/circle-question-regular'` we could do something like `icon.question`. Thoughts?

Reply:
I'm interested in the idea, definitely one for stand-up or collab time